runbotics/runbotics-desktop/src/action/microsoft/excel/excel.service.ts

```diff
@@ -219,6 +235,16 @@ export class ExcelService {
         return (column.length - 1) * 26 + (column.charCodeAt(column.length - 1) - 64);
     }
 
+    /**
+     * @description converts column number to a letter format
+     * @param {number} columnNumber column number to convert
+     * @returns {string} column in a letter format
+     */
+    private getColumnLetter(columnNumber: number): string {
+        const letter = 'ABCDEFGHIJKLMNOPQRSTUVWXYZ'[(columnNumber - 1) % 26];
+        return columnNumber >= 26 ? this.getColumnLetter(Math.floor(columnNumber / 26) - 1) + letter : letter;
```

Comment:
shouldn't be there columnNumber > 26? Now i'm counting for getColumnLetter(26) // expected: 'Z' got: 'ZA'

Reply:
thanks! :)